packages/framework/src/Form/Admin/Country/CountryFormType.php

```diff
@@ -30,19 +29,12 @@ class CountryFormType extends AbstractType
      */
     protected $countryFacade;
 
-    /**
```

Comment:
This should not be part of the last commit

Reply:
You're absolutely right. I noticed it while writing the test. I can move it on rebase.